sig-docs/localization-subproject/README.md

```diff
@@ -0,0 +1,105 @@
+# Localization Subproject
+
+The Kubernetes Localization Subproject is owned by [SIG Docs](https://github.com/kubernetes/community/tree/master/sig-docs) and run by the [Subproject leads](#leadership).
+
+The Kubernetes documentation has been localized into several different languages. Each localization is owned by a team of individuals that speaks the language.  A localization team is a self-sufficient community responsible for prioritizing work, translating content, and reviewing and merging PRs into the kubernetes/website repo for their localization. 
+
+The Localization Subproject's aim is to provide a place for localization teams to share ideas and standardize processes across all localizations. This page covers documentation, processes, and roles for the Kubernetes Localization teams and the Localization Subproject.
+
+## Meetings
+
+Regular Localization Meeting: The first Monday of the month at 15:00 UTC (biweekly). [Convert it to your timezone](http://www.thetimezoneconverter.com/?t=15:00&tz=UTC)
+
+- [Meeting notes and Agenda](https://docs.google.com/document/d/1NwO1AN8Ea2zlK8uAdaDAKf1-LZDAFvSewIfrKqfl5No/edit#)
+- [Meeting recordings](https://www.youtube.com/playlist?list=PL69nYSiGNLP3b5hlx0YV7Lo7DtckM84y8)
+
+A localization team may also hold language specific meetings. If you are interested in attending a language specific meeting see the [Localization Teams](#localization-teams) section for details on how to contact each localization team.
+
+## Subproject Leadership
+
+- **Subproject lead:** [Abigail McCarthy](https://github.com/a-mccarthy)
+
+- **Emeritus leads:** [Brad Topol](https://github.com/bradtopol)
+
+## Contact
+
+- Slack: [#sig-docs-localizations](https://kubernetes.slack.com/archives/C0191RDKHU1) Each localization also has a language specific channel, noted [below](#localization-teams). 
+- Mailing list: https://groups.google.com/g/kubernetes-sig-docs-localization
+
+## Responsibilities
+
+The Localization Subproject lead is **responsible** for
+
+- Leading Subproject meetings and taking notes.
+- Driving collaborative discussions and decisions around localization processes and standards.
+- Ensuring the communication of localization and SIG Docs policy changes to localization teams.
+- Advocating for localization-friendly practices within SIG Docs and the broader Kubernetes community.
+- Helping Localization teams understand SIG Docs processes, especially when languages are starting out
+- Making sure that [Localization process documentation](https://kubernetes.io/docs/contribute/localization/) is kept up to date.
+
+The Localization Subproject lead is **not responsible** for
+
+- Running localization efforts for a language
+- Dictating how localization teams do work or organize
+- Making decisions for localizations policies or teams without the localization team's input
+
+### Skills and Experience Required 
```

Comment:
IMHO it does not make sense to have 3rd heading section, if there is only one on that level. 
```suggestion
## Skills and Experience Required 
```